app/views/newflow/login_signup/_password_show_hide.html.erb

```diff
@@ -1,6 +1,10 @@
 <span id="password-show-hide-button">
-    <span class="toggle-show-hide">SHOW</span>
-    <span class="toggle-show-hide" style="display: none">HIDE</span>
+    <span class="toggle-show-hide">
+      <%= I18n.t(:"login_signup_form.show_password_btn")  %>
+    </span>
+    <span class="toggle-show-hide" style="display: none">
```

Comment:
do we want style the erb?  this seems kinda controversial.  just asking.

Reply:
If I understand your question correctly, yes, that is intentional. I don't know how else to make that work.

Reply:
unobtrusive css :)   to start with a class that denotes hidden.